lib/lti/variable_expander.rb

```diff
@@ -516,6 +518,15 @@ def enabled_capability_params(enabled_capabilities)
                        -> { @context.enrollment_term.start_at },
                        TERM_START_DATE_GUARD
 
+    # returns the current course's term name.
+    # @example
+    #   ```
+    #   W1 2017
+    #   ```
+    register_expansion 'Canvas.term.name', [],
+                        -> { @context.enrollment_term.name },
+                        TERM_NAME_GUARD
+
```

Comment:
Let's also add a `default_name` for this substitution:
```
...
    register_expansion 'Canvas.term.name', [],
                        -> { @context.enrollment_term.name },
                        TERM_NAME_GUARD,
                         default_name: 'canvas_term_name'
...

This will allow us to use this variable expansion with LTI 2.x tools in an easier way as well.
```

Reply:
added